config/settings.yml

```diff
@@ -0,0 +1,41 @@
+project_groups:
```

Comment:
We should move everything from `constants.rb` here IMHO.

Reply:
Definitely agree we should consolidate. But concerned about this interim duplication since it becomes even more fragile if we ever need to create/edit a project type in the future. Perhaps better to pull the bandaid now and do it the right way?